src/storage/src/hummock/test_utils.rs

```diff
@@ -92,21 +94,29 @@ pub fn gen_test_sstable_data(
 }
 
 /// Generates a test table from the given `kv_iter` and put the kv value to `sstable_store`
-pub async fn gen_test_sstable(
+pub async fn gen_test_sstable_inner(
     opts: SSTableBuilderOptions,
     sst_id: u64,
     kv_iter: impl Iterator<Item = (Vec<u8>, HummockValue<Vec<u8>>)>,
     sstable_store: SstableStoreRef,
+    poliy: CachePolicy,
 ) -> Sstable {
     let (data, meta) = gen_test_sstable_data(opts, kv_iter);
     let sst = Sstable { id: sst_id, meta };
-    sstable_store
-        .put(&sst, data, CachePolicy::Fill)
-        .await
-        .unwrap();
+    sstable_store.put(&sst, data, poliy).await.unwrap();
     sst
 }
 
+/// Generate a test table from the given `kv_iter` and put the kv value to `sstable_store`
+pub async fn gen_test_sstable(
+    opts: SSTableBuilderOptions,
+    sst_id: u64,
+    kv_iter: impl Iterator<Item = (Vec<u8>, HummockValue<Vec<u8>>)>,
+    sstable_store: SstableStoreRef,
+) -> Sstable {
+    gen_test_sstable_inner(opts, sst_id, kv_iter, sstable_store, CachePolicy::Fill).await
+}
+
```

Comment:
Do we really need the wrapper? 🤔

Reply:
in this test, we need `CachePolicy::NotFill`,  So i use the wrapper (Most of the calling methods use the `CachePolicy::Fill`)